CleverPush/Source/CPSQLiteManager.m

```diff
@@ -0,0 +1,245 @@
+#import "CPSQLiteManager.h"
+#import "CleverPush.h"
+#import "CPLog.h"
+
+@implementation CPSQLiteManager
+
+static CPSQLiteManager *sharedInstance = nil;
+NSString *databaseTable = @"cleverpush_table_banner_track_event";
```

Comment:
Please use same table name + same column names like in Android PR

(I think table e.g. is called cleverpush_tracked_events)

Reply:
@cpandya25